tools/reset_github_user_token_and_update_circleci.sh

```diff
@@ -17,7 +16,7 @@ then
     exit 1
 fi
 
-BROWSER_NAME=firefox
+BROWSER_NAME=Firefox
 BROWSER="/Applications/Firefox.app/Contents/MacOS/firefox -private-window"
```

Comment:
??

How Firefox and MacOS path ended up in automation script like that?

Reply:
@blag is the only person that uses this script. As such, it was designed for his setup using Firefox on Mac. I put these in vars so that we might be able to prompt for which browser to use at some future point, if anyone else on the TSC volunteers to help @blag maintain this part of the Exchange.

Reply:
To clarify further: There are no APIs that we can use to generate PATs. Someone has to manually generate the token in the Github web ui. However, we can prefill the form and open a browser window to save as many steps as possible in this manual process.
We use private/incognito browsing because the tokens need to be linked to a bot account that is a member of the StackStorm-Exchange organization, not a personal account. Private/Incognito simplifies keeping that account separate from a personal account.

Reply:
Fixed this.